### PR TITLE
Prevent crashing when API has camelcase namespaces

### PR DESCRIPTION
**Pull request by gekola**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 * Your contribution here.
 
+#### Fixes
+
+* [#321](https://github.com/ruby-grape/grape-swagger/pull/321): Fixed handling pathes containing uppercase letters - [@gekola](https://github.com/gekola)
```
Comment (dblock) [inline CHANGELOG.md]: Missing a period at the end :)

Comment (dblock) [inline CHANGELOG.md]: Pathes => paths.

**Comment by dblock** [comment]
I think this makes sense, but I wonder why this was being downcased before, did you track down the origin of that code?

**Comment by dblock** [comment]
Also, what "crashing" does this prevent?

**Comment by gekola** [comment]
With the latest version I get
```.../vendor/bundle/ruby/2.2.0/gems/grape-swagger-0.10.4/lib/grape-swagger.rb:80:in `block in combine_namespace_routes': undefined method `collect' for nil:NilClass (NoMethodError)```
on application load. The namespace was put in downcase to the `@target_class.combined_routes`.

The `downcase!` was introduced in 033bc8013426b774da553375315f1d1ef1c016d0. There are no specific tests added for the case and no comments that indicate it is required by any other code.

**Comment by dblock** [comment]
:+1: merging (the build is failing on rbx, different story)

**Pull request merged**